src/app/application.cpp

```diff
@@ -465,6 +477,7 @@ void Application::cleanup()
     Net::DownloadManager::freeInstance();
     Preferences::freeInstance();
     Logger::freeInstance();
+    delete m_fileLogger;
```

Comment:
Yes, but it never accesses logger directly, only in construction to setup the signal/slots. My intention was to leave it last in order to catch all messages to file.
Do you still want to move it?